R/annotation.R

```diff
@@ -266,20 +266,20 @@ annotate_driver_ssm = function(maf_df,
 #' annotated_sv = annotate_sv(sv_df)
 #' annotated_entrez = annotate_sv(sv_data = sv_df, with_chr_prefix = FALSE, collapse_redundant = FALSE, return_as = "bedpe_entrez", genome_build = "grch37")
 #'
-annotate_sv = function(sv_data, 
-                       partner_bed, 
-                       with_chr_prefix = FALSE, 
-                       collapse_redundant = FALSE, 
+annotate_sv = function(sv_data,
+                       partner_bed,
+                       with_chr_prefix = FALSE,
+                       collapse_redundant = FALSE,
                        return_as = "bedpe_entrez",
-                       blacklist = c(60565248, 30303126, 187728894, 101357565, 101359747, 161734970, 69400840, 65217851, 187728889, 188305164),
+                       blacklist = c(60565248, 30303126, 187728894, 101357565, 101359747, 161734970, 69400840, 65217851, 187728889, 187728888,187728892, 187728893,188305164),
```

Comment:
Should this blacklist parameter I implemented be pointing to a bed file instead? Or is it enough, for now, to just state such regions like this?

Reply:
We should definitely move this information to a bundled file